sysl2/sysl/codegen.go

```diff
@@ -231,10 +231,14 @@ func GenerateCode(root_model, model, root_transform, transform, grammar, start s
 
 	mod, modelAppName := loadAndGetDefaultApp(root_model, model)
 	tx, transformAppName := loadAndGetDefaultApp(root_transform, transform)
+	grm, _ := loadAndGetDefaultApp("grammars", "go.gen.sysl")
```

Comment:
How will it work with other grammar models?

Reply:
My bad. I fixed it.